.github/workflows/image.yml

```diff
@@ -2,67 +2,120 @@ on:
   pull_request:
   push:
     branches: [master]
+
 jobs:
   build:
-    outputs:
-      image_tag: ${{ steps.get_image_tag.outputs.image_tag }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-24.04
+            platform: amd64
```

Comment:
Can we run amd64 images natively on arm64 without using emulation?

Reply:
No, tried it before, it sucked.